docs/production/deploying-to-cloud/gcp/setup.mdx

```diff
@@ -178,24 +89,20 @@ variable "project_id" {
 }
 ```
 
-**Docker image (REQUIRED)**
+**Mage Image (REQUIRED)**
```

Comment:
The variable is called `docker_image`. So I'd prefer still using "Docker image" as the title here or "Mage docker image".